lib/l10n/app_en.arb

```diff
@@ -193,10 +193,93 @@
+  "pageHelpQuestion3": "We agreed on a price, but now my rider isn't paying it. What can I do?",
+  "pageHelpAnswer3": "We're sorry for the inconvenience. Unfortunately, the only thing we can offer you is the ability to report users in that case. Tap the \"{reportButtonText}\" button on their profile and make sure to select the reason \"{didNotPayReasonText}\".",
+  "@pageHelpAnswer3": {
+    "placeholders": {
+      "reportButtonText": {},
+      "didNotPayReasonText": {}
+    }
+  },
+  "pageHelpQuestion4": "A driver/rider didn't show up. What can I do?",
+  "pageHelpAnswer4": "We're sorry for the inconvenience. Unfortunately, the only thing we can offer you is the ability to report users in that case. Tap the \"{reportButtonText}\" button on their profile and make sure to select the reason \"{didNotShowUpReasonText}\".",
+  "@pageHelpAnswer4": {
+    "placeholders": {
+      "reportButtonText": {},
+      "didNotShowUpReasonText": {}
+    }
+  },
+  "pageHelpQuestion5": "My ride got cancelled. What can I do?",
+  "pageHelpAnswer5": "We're sorry for the inconvenience. Unfortunately, you will have to look for another ride. If the driver cancelled the drive just before the trip, you can report them using the \"{reportButtonText}\" button on their profile. Make sure to select the reason \"{otherReasonText}\" and explain the situation.",
+  "@pageHelpAnswer5": {
+    "placeholders": {
+      "reportButtonText": {},
+      "otherReasonText": {}
+    }
+  },
+  "pageHelpQuestion6": "My ride was rejected. What can I do?",
+  "pageHelpAnswer6": "We're sorry for the inconvenience. Unfortunately, you will have to look for another ride.",
+  "pageHelpQuestion7": "The driver isn't reacting to my request. What can I do?",
+  "pageHelpAnswer7": "We're sorry for the inconvenience. If you tap on the drive, you can take the request back using the \"{withdrawButtonText}\" button at the bottom. Then, you can look for a new ride.",
+  "@pageHelpAnswer7": {
+    "placeholders": {
+      "withdrawButtonText": {}
+    }
+  },
+  "pageHelpQuestion8": "My driver/riders are misbehaving. What can I do?",
+  "pageHelpAnswer8": "We're sorry for the inconvenience. We are not responsible for the actions of our users. If other users are using bad language or behaving aggressively, offensively or otherwise inappropriately, please report them using the \"{reportButtonText}\" button on their profile. Make sure to select the appropriate reason.",
+  "@pageHelpAnswer8": {
+    "placeholders": {
+      "reportButtonText": {}
+    }
+  },
+  "pageHelpQuestion9": "Where can I edit my profile?",
+  "pageHelpAnswer9": "You can tap on your user name at the top of the \"{accountTabText}\" tab.",
+  "@pageHelpAnswer9": {
+    "placeholders": {
+      "accountTabText": {}
+    }
+  },
+  "pageHelpQuestion10": "Where can I offer drives?",
+  "pageHelpAnswer10": "You can tap on the \"+\" icon at the bottom right of the \"{drivesTabText}\" tab.",
```

Comment:
```suggestion
  "pageHelpAnswer10": "You can tap on the \"+\" icon at the bottom right of the \"{drivesTabText}\" tab or click on the \"{create Drive }\" button on the Homepage.",
```
same for rides

Reply:
Let's wait with that until home is done, and we can use the translation of the button in this string. Otherwise that could get out of sync.